src/Site.js

```diff
@@ -922,7 +923,7 @@ class Site {
     injectMarkdownItSpecialTags(tagsToIgnore);
     Page.htmlBeautifyOptions = {
       indent_size: 2,
-      content_unformatted: ['pre', ...tagsToIgnore],
+      content_unformatted: ['pre', 'textarea', ...tagsToIgnore],
```

Comment:
What's the rationale for adding this new `textarea` option?

Reply:
html beautify was updated with this as well since our last version; unfortunately the options don't merge with the defaults, so a little manual inspection has to be done here

https://github.com/beautify-web/js-beautify/blob/master/js/src/html/options.js